src/Collection/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { getCurrentNetworkURNProtocol } from '../ethereum/utils'
+import { CollectionAttributes, FullCollection } from './Collection.types'
+
+export function getDecentralandCollectionURN(
+  collectionAddress: string
+): string {
+  return `urn:decentraland:${getCurrentNetworkURNProtocol()}:collections-v2:${collectionAddress}`
+}
+
+export function toFullCollection(
+  dbCollection: CollectionAttributes
+): FullCollection {
+  const fullCollection: FullCollection = {
+    ...dbCollection,
+    urn: getDecentralandCollectionURN(dbCollection.contract_address),
+  }
+  delete (fullCollection as FullCollection & { urn_suffix: unknown }).urn_suffix
```

Comment:
we could take advantage of `decentraland-common's` omit here if you like. Avoiding this `delete` for something like
```ts
  const fullCollection: FullCollection = {
    ...utils.omit(dbCollection, ['urn_suffix']),
    urn: getDecentralandCollectionURN(dbCollection.contract_address),
  }
```

or just

```ts
  const { urn_suffix, ...sufixlessDbCollection } = dbCollection
  const fullCollection: FullCollection = {
    ...sufixlessDbCollection,
    urn: getDecentralandCollectionURN(dbCollection.contract_address),
  }
```
but that might upset the linter

Reply:
Same goes for `toDBCollection` with the urn!

Reply:
Done!